lib/Model/Listing/Dao/QueryBuilderHelperTrait.php

```diff
@@ -29,19 +28,12 @@ trait QueryBuilderHelperTrait
      */
     protected $onCreateQueryBuilderCallback;
 
-    /**
-     * @param callable|null $callback
-     */
```

Comment:
why removed the phpdocs?

Reply:
Redundant as it contains exactly the same information as already stated in the method's signature. I'm actually used to remove such duplicate info (within our company it's best practice to do so), so probably did this from muscle memory. Happy to revert this if you prefer

Reply:
I got your point now. Let's keep it this way 😉